MagicBots/Overseer/Overseer.csproj

```diff
@@ -33,8 +34,4 @@
     </None>
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="Modules\" />
```

Comment:
did you mean to remove this?

Reply:
Yes, there's nothing in that folder right now we can readd it when there is actually something there. It shows up as an error in my IDE.